kolibri/core/errorreports/serializers.py

```diff
@@ -1,6 +1,12 @@
 from rest_framework import serializers
 
+from .schemas import context_frontend_schema
+from kolibri.core.utils.validators import JSON_Schema_Validator
+
 
 class ErrorReportSerializer(serializers.Serializer):
     error_message = serializers.CharField(max_length=255)
     traceback = serializers.CharField()
+    context = serializers.JSONField(
```

Comment:
Note, I think if you just use a DRF model serializer for the model, most of those configuration should be automatic. https://www.django-rest-framework.org/api-guide/serializers/#modelserializer